app/resources/game_content.py

```diff
@@ -0,0 +1,203 @@
+    m.contact_microservice("inventory", ["inventory", "delete_by_name"], {"owner": user, "item_name": elements["cpu"]})
+    m.contact_microservice(
+        "inventory", ["inventory", "delete_by_name"], {"owner": user, "item_name": elements["motherboard"]}
+    )
+    m.contact_microservice("inventory", ["inventory", "delete_by_name"], {"owner": user, "item_name": elements["gpu"]})
+    for ram in elements["ram"]:
+        m.contact_microservice("inventory", ["inventory", "delete_by_name"], {"owner": user, "item_name": ram})
+    for disk in elements["disk"]:
+        m.contact_microservice("inventory", ["inventory", "delete_by_name"], {"owner": user, "item_name": disk})
+
+
+def check_elements_exist(elements: dict) -> Tuple[bool, dict]:
+    if not elements["cpu"] in hardware["cpu"].key():
+        return False, {"error": "element_cpu_not_found"}
+    if not elements["gpu"] in hardware["gpu"].keys():
+        return False, {"error": "element_gpu_not_found"}
+    if not elements["motherboard"] in hardware["mainboards"].keys():
+        return False, {"error": "element_motherboard_not_found"}
+    for disk in elements["disk"]:
+        if disk not in hardware["diskStorage"].keys():
+            return False, {"error": "element_disk_not_found"}
+    for ram in elements["ram"]:
+        if ram not in hardware["ram"].keys():
+            return False, {"error": "element_ram_not_found"}
+
+    return True, {}
+
+
+def check_compatible(elements: dict) -> Tuple[bool, dict]:
+    exists, message = check_elements_exist(elements)
+    if not exists:
+        return False, message
+
+    cpu: str = elements["cpu"]
+    motherboard: str = elements["motherboard"]
+    ram: List[str] = elements["ram"]
+    gpu: str = elements["gpu"]
+    disk: List[str] = elements["disk"]
+
+    if hardware["cpu"][cpu]["sockel"] != hardware["mainboards"][motherboard]["sockel"]:
+        return False, {"error": "cpu_and_mainboard_sockel_do_not_fit"}
+
+    if hardware["mainboards"][motherboard]["ram"]["ramSlots"] < len(ram):
+        return False, {"error": "mainboard_has_not_this_many_ram_slots"}
+
+    for ram_stick in ram:
+        if hardware["ram"][ram_stick]["ramTyp"] != hardware["mainboards"][motherboard]["ram"]["typ"]:
+            return False, {"error": "ram_type_does_not_fit_what_you_have_on_your_mainboard"}
+
+    for i in disk:
+        if hardware["diskStorage"][i]["interface"] != hardware["mainboards"][motherboard]["diskStorage"]["interface"]:
+            return False, {"error": "your_hard_drive_interface_does_not_fit_with_the_motherboards_one"}
+
+    return True, {}
+
+
+def calculate_power(elements: dict) -> Tuple[float, float, float, float, float]:
+    cpu: str = elements["cpu"]
+    motherboard: str = elements["motherboard"]
+    ram: List[str] = elements["ram"]
+    gpu: str = elements["gpu"]
+    disk: List[str] = elements["disk"]
+
+    performance_cpu: float = hardware["cpu"][cpu]["cores"] * hardware["cpu"][cpu]["frequencyMax"]
```

Comment:
`hardware["cpu"]` does not exist
```suggestion
    performance_cpu: float = hardware["processor"][cpu]["cores"] * hardware["processor"][cpu]["frequencyMax"]
```

Reply:
renamed keys done